charts/argocd-apps/values.yaml

```diff
@@ -40,6 +40,46 @@ applications: []
 #   - name: url
 #     value: https://argoproj.github.io/
 
+applicationTemplates:
+  []
```

Comment:
Please follow the style we currently use and put the brackets on the same line as the key:

```yaml
myList: []
myDictionary: {}
```